cmd/goexports/goexports.go

```diff
@@ -212,7 +222,7 @@ func genContent(dest, pkgName string) ([]byte, error) {
 		buildTags += ",!windows,!nacl,!plan9"
 	}
 
-	b := &bytes.Buffer{}
+	b := new(bytes.Buffer)
```

Comment:
could you explain this change?

Reply:
it's usually spelled this way in Go's stdlib.